docs/make.jl

```diff
@@ -1,3 +1,15 @@
-using Documenter, GLM
+using Distributions, Documenter, GLM
 
-makedocs()
+makedocs(
+    format = :html,
```

Comment:
I think you can remove `docs/mkdocs.yml` with that option. And of course the old `docs/` folder.

Reply:
Done, thanks.